src/components/CommandBar/CommandArgOptionInput.tsx

```diff
@@ -175,9 +177,16 @@ function CommandArgOptionInput({
             <Combobox.Option
               key={option.name}
               value={option}
+              disabled={option.disabled}
               className="flex items-center gap-2 px-4 py-1 first:mt-2 last:mb-2 ui-active:bg-primary/10 dark:ui-active:bg-chalkboard-90"
             >
-              <p className="flex-grow">{option.name} </p>
+              {/* TODO: change opacity hack to a real class */}
+              <p
+                className="flex-grow"
+                style={{ opacity: option.disabled ? 0.5 : 1.0 }}
+              >
```

Comment:
@franknoirot do you have something better here? Tried to use some disabled text class I found elsewhere but couldn't get it to work